recipes/genozip/meta.yaml

```
@@ -0,0 +1,70 @@
+
```

Using a version variable minimises mistakes and means the bot can auto update:
```suggestion
{% version = "1.0.1" %}
```